composer.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "justbetter/magento2-sentry-filter-events",
+  "description": "Magento 2 Filter for Sentry Events",
+  "type": "magento2-module",
+  "version": "1.0.0",
```

Comment:
It's better to remove this so packagist will listen to the tag attached to the commit
```suggestion
```